### PR TITLE
Implemented captcha for building form

### LINKED ISSUE #670: Add CAPTCHA verification for the add building form
Before submitting a new building, a user should be given a CAPTCHA to complete.

After looking at multiple solutions, hCAPTCHA seems to be a better solution for this task. 
hCAPTCHA seems to be more privacy-focused without compromises on performance or the ease of implementation.
🔗 [hCAPTCHA documentation](https://docs.hcaptcha.com/)

### PR DESCRIPTION
**Pull request by damianr13**
Closes #670  

Added dependency for hCaptcha module and included it in the form. 

All the tested has been done manually.

### CODE REVIEW

File: client/src/containers/add-building/FormFragment.js [hunk 1]
```diff
@@ -187,6 +192,11 @@ const FormFragment = ({ form }) => {
           </Select>,
         )}
       </Form.Item>
+      <Form.Item label={<Trans>Captcha</Trans>}>
+        {getFieldDecorator('captcha', {
+          rules: [{ required: true, message: <EmptyFieldMessage /> }],
+        })(<HCaptcha sitekey="your-sitekey" onVerify={handleVerifyCaptcha} />)}
```
Comment (tudoramariei) [inline client/src/containers/add-building/FormFragment.js:198]: We will need to pass the key the same way it's done for the `MAP_API_KEY` (`REACT_APP_API_KEY` in `.env.dist`)

**Comment by tudoramariei** [comment]
Please also fix the issue found by the `Client Pipeline` run
(remove the [console.log statement](https://github.com/code4romania/seismic-risc/pull/679/files/62f954bebc439cb9c71eca7322424a171c0f2aa0#diff-59e0852e15f7ae4f0460e56839fade1ced1aa6d69b18c55d08dbd6708ea9e37aR72))

File: client/src/containers/add-building/FormFragment.js [hunk 2]
```diff
@@ -67,6 +68,9 @@ const FormFragment = ({ form }) => {
     setCoordinates(newCoordinates);
   };
 
+  // eslint-disable-next-line no-unused-vars
+  const handleVerifyCaptcha = (token, eKey) => {};
```
Comment (surdu) [inline client/src/containers/add-building/FormFragment.js:72]: Don't we have to do something with the token ?

**Review by surdu — CHANGES_REQUESTED** [review]

File: client/src/containers/add-building/FormFragment.js (hunk 2 above)
Comment (tudoramariei) [reply client/src/containers/add-building/FormFragment.js:72]: I think the method should somehow check that once the captcha was verified the form can be submitted.
Right now it seems that, [in the preview, submitting a new building](https://seismic-risc-rlamg8au2-code4romania.vercel.app/adauga-cladire) isn't allowed even after completing the captcha 

Screenshot 2021-06-29 at 18 25 16

**Review by tudoramariei — APPROVED** [review]

**Pull request merged**